homeassistant/components/overkiz/climate_entities/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,180 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+from __future__ import annotations
+
+from typing import cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.const import TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+FAN_BOOST = "home_boost"
+FAN_KITCHEN = "kitchen_boost"
+FAN_AWAY = "away"
+FAN_BYPASS = "bypass_boost"
+
+PRESET_AUTO = "auto"
+PRESET_PROG = "prog"
+PRESET_MANUAL = "manual"
+
+OVERKIZ_TO_FAN_MODES: dict[str, str] = {
+    OverkizCommandParam.AUTO: FAN_AUTO,
+    OverkizCommandParam.AWAY: FAN_AWAY,
+    OverkizCommandParam.BOOST: FAN_BOOST,
+    OverkizCommandParam.HIGH: FAN_KITCHEN,
+    "": FAN_BYPASS,
+}
+
+FAN_MODES_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_FAN_MODES.items()}
+
+TEMPERATURE_SENSOR_DEVICE_INDEX = 4
+
+
+class AtlanticHeatRecoveryVentilation(OverkizEntity, ClimateEntity):
+    """Representation of a AtlanticHeatRecoveryVentilation device."""
+
+    _attr_fan_modes = [*FAN_MODES_TO_OVERKIZ]
+    _attr_hvac_modes = [HVACMode.FAN_ONLY]
```

Comment:
```suggestion
    _attr_hvac_mode = HVACMode.FAN_ONLY
    _attr_hvac_modes = [HVACMode.FAN_ONLY]
```

Reply:
Thanks, good catch! Fixed in 485ef31a8e37311bcba90f72127fb857f8695aeb